pom.xml

```diff
@@ -1402,7 +1402,31 @@
             <dependency>
                 <groupId>org.wildfly.security.elytron-web</groupId>
                 <artifactId>undertow-server</artifactId>
-                <version>${version.org.wildfly.security.elytron-web.undertow-server}</version>
+                <version>${version.org.wildfly.security.elytron-web}</version>
+                <exclusions>
+                    <exclusion>
+                        <artifactId>*</artifactId>
+                        <groupId>*</groupId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.wildfly.security.elytron-web</groupId>
+                <artifactId>undertow-server-servlet</artifactId>
+                <version>${version.org.wildfly.security.elytron-web}</version>
+                <exclusions>
+                    <exclusion>
+                        <artifactId>*</artifactId>
+                        <groupId>*</groupId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.wildfly.security.elytron-web</groupId>
+                <artifactId>undertow-server-servlet</artifactId>
```

Comment:
Is there a reason this is defined twice?

Reply:
Looks like a rebase error, will remove the duplicate.